studio-docs/source/metrics/operation-signatures.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Operation signatures
+description: How Studio identifies equivalent operations
+---
+
+In your metrics reports, Apollo Studio groups together operations based on the exact set of fields they include, _not_ based on operation _names_. This means that operations with the same name but different sets of fields are displayed separately:
+
+<img src="../img/separate-operations.jpg" alt="Separate operations with same name in Studio" class="screenshot" width="500" />
+
+To help Studio identify functionally identical operations, Apollo Server generates the **signature** for each operation it reports to Studio. This signature is a normalized representation for an operation with deterministic field order, whitespace, and values for in-line argument values.
```

Comment:
I think the main thing these docs need to make clear is that this is for informational purposes, not for normative purposes, and that we may change details of it later! I also don't think you have 100% listed every place that sortAST applies sorting to the tree, though I don't know if that's really an achievable goal.